packages/datatrak-web-server/src/routes/TasksRoute.ts

```diff
@@ -41,79 +40,128 @@ type SingleTask = Task & {
   'entity.country_code': string;
 };
 
-type FormattedFilter =
-  | string
-  | { comparator: string; comparisonValue: string; castAs?: string }
-  | TaskStatus;
+type FormattedFilters = Record<string, any>;
 
-type FormattedFilters = Record<string, FormattedFilter>;
+const EQUALITY_FILTERS = ['due_date', 'survey.project_id', 'task_status'];
 
-const queryForCount = async (filter: FormattedFilters, models: DatatrakWebServerModelRegistry) => {
-  const filtersWithColumnSelectors = { ...filter };
+const getFilterSettings = (cookieString: string) => {
+  const cookies = parse(cookieString);
+  return {
+    allAssignees: cookies['all_assignees_tasks'] === 'true',
+    allCompleted: cookies['all_completed_tasks'] === 'true',
+    allCancelled: cookies['all_cancelled_tasks'] === 'true',
```

Comment:
I'm wondering if `all_cancelled...` and `all_completed...` makes sense, since it's just whether to include them in the results, versus returning them all